File: backend/Models/Event.cs
namespace TodoApi.Models;

public class Event
{
    public long Id { get; set; }
    public required string Title { get; set; }
    public required string Description { get; set; }
    public string? Image { get; set; }
    public DateTime? StartDate { get; set; }
    public DateTime? EndDate { get; set; }
    public ICollection<Attendee> Attendees { get; set; } = new List<Attendee>();
}

File: frontend/src/models/Event.ts
import type { Attendee } from "../../components/Events/EventDrawer/types";
import type { Review } from "./Review";

export interface Event {
	id: number;
	title: string;
	description: string;
	image?: string;
	startDate?: string;
	endDate?: string;
	attendees?: Attendee[];
	reviews?: Review[];
}

export interface CreateEventRequest {
	Title: string;
	Description: string;
	Image?: string;
	StartDate?: string;
	EndDate?: string;
}
